lib/fauna.rb

```diff
@@ -9,6 +9,7 @@
 # Main namespace for the FaunaDB client.
 module Fauna; end
 
+require 'fauna/_json'
```

Comment:
Does calling it `_json` actually effect anything that would prevent external things from using it? AFAIK when people import the client, the `Fauna::FaunaJson` module is coming along for the ride and will be accessible.